tests/manage/pv_services/test_change_reclaim_policy_of_pv.py

```diff
@@ -77,9 +77,12 @@ def setup(
                 pod_factory(
                     interface=interface,
                     pvc=pvc_obj,
-                    status=constants.STATUS_RUNNING
+                    status=False
```

Comment:
status gets string and here you pass bool. Shouldn't we pass None instead?

Reply:
Correct.
Fix in next PR